universal-login-relayer/src/http/routes/email.ts

```diff
@@ -2,12 +2,14 @@ import {Router} from 'express';
 import {asyncHandler, sanitize, responseOf} from '@restless/restless';
 import {asString, asObject} from '@restless/sanitizers';
 import {EmailConfirmationHandler} from '../../core/services/EmailConfirmationHandler';
-import {asSerializableRequestedCreatingWallet, SerializableRequestedCreatingWallet} from '@unilogin/commons';
+import {asSerializableRequestedCreatingWallet, SerializableRequestedCreatingWallet, ensure} from '@unilogin/commons';
+import {AlreadyUsed} from '../../core/utils/errors';
 
-const emailConfirmationRequest = (emailConfirmationHandler: EmailConfirmationHandler) =>
+const emailConfirmationRequest = (handler: EmailConfirmationHandler) =>
   async (data: {body: SerializableRequestedCreatingWallet}) => {
     const {email, ensName} = data.body;
-    const result = await emailConfirmationHandler.request(email, ensName);
+    ensure(await handler.emailConfirmationStore.getConfirmedNumber(email, ensName) === 0, AlreadyUsed, `${email} or ${ensName}`);
```

Comment:
let's make it like this: create function in handler: 
```ts
isUniqueRequest() {
     await this..emailConfirmationStore.getConfirmedNumber(email, ensName) === 0
}
```

and use it in `request` function like this:
```ts
request() {
    ensure(this.isUniqueRequest(), AlreadyUsed)
    ...
}
```